pkg/scheduler/actions/enqueue/enqueue_test.go

```diff
@@ -0,0 +1,146 @@
+				util.BuildPod("c1", "p2", "", v1.PodPending, api.BuildResourceList("1", "1G"), "pg1", make(map[string]string), make(map[string]string)),
+			},
+			Queues: []*schedulingv1.Queue{
+				util.BuildQueue("c1", 1, api.BuildResourceList("4", "4G")),
+				util.BuildQueue("c2", 1, api.BuildResourceList("4", "4G")),
+			},
+			Status: map[api.JobID]scheduling.PodGroupPhase{
+				"c1/pg1": scheduling.PodGroupInqueue,
+			},
+		},
+		{
+			Name: "when podgroup status is pending",
+			PodGroups: []*schedulingv1.PodGroup{
+				util.BuildPodGroup("pg1", "c1", "c1", 0, nil, schedulingv1.PodGroupPending),
+				util.BuildPodGroup("pg2", "c1", "c2", 0, nil, schedulingv1.PodGroupPending),
+			},
+			Pods: []*v1.Pod{
+				// pending pod with owner1, under ns:c1/q:c1
+				util.BuildPod("c1", "p1", "", v1.PodPending, api.BuildResourceList("3", "1G"), "pg1", make(map[string]string), make(map[string]string)),
+				// pending pod with owner2, under ns:c1/q:c2
+				util.BuildPod("c1", "p2", "", v1.PodPending, api.BuildResourceList("1", "1G"), "pg2", make(map[string]string), make(map[string]string)),
+			},
+			Queues: []*schedulingv1.Queue{
+				util.BuildQueue("c1", 1, api.BuildResourceList("4", "4G")),
+				util.BuildQueue("c2", 1, api.BuildResourceList("4", "4G")),
+			},
+			Status: map[api.JobID]scheduling.PodGroupPhase{
+				"c1/pg1": scheduling.PodGroupInqueue,
+				"c1/pg2": scheduling.PodGroupInqueue,
+			},
+		},
+		{
+			Name: "when podgroup status is running",
+			PodGroups: []*schedulingv1.PodGroup{
+				util.BuildPodGroup("pg1", "c1", "c1", 0, nil, schedulingv1.PodGroupRunning),
+			},
+			Pods: []*v1.Pod{
+				util.BuildPod("c1", "p1", "", v1.PodRunning, api.BuildResourceList("1", "1G"), "pg1", make(map[string]string), make(map[string]string)),
+				util.BuildPod("c1", "p2", "", v1.PodRunning, api.BuildResourceList("1", "1G"), "pg1", make(map[string]string), make(map[string]string)),
+			},
+			Queues: []*schedulingv1.Queue{
+				util.BuildQueue("c1", 1, api.BuildResourceList("4", "4G")),
+			},
+			Status: map[api.JobID]scheduling.PodGroupPhase{
+				"c1/pg1": scheduling.PodGroupRunning,
+			},
+		},
+		{
+			Name: "pggroup cannot enqueue because the specified queue is c1, but there is only c2",
+			PodGroups: []*schedulingv1.PodGroup{
+				util.BuildPodGroup("pg1", "c1", "c1", 0, nil, schedulingv1.PodGroupPending),
+			},
+			Queues: []*schedulingv1.Queue{
+				util.BuildQueue("c2", 1, api.BuildResourceList("4", "4G")),
+			},
+			Status: map[api.JobID]scheduling.PodGroupPhase{},
+		},
+		{
+			Name: "pggroup cannot enqueue because queue resources are less than podgroup MinResources",
+			PodGroups: []*schedulingv1.PodGroup{
+				util.BuildPodGroupWithMinResources("pg1", "c1", "c1", 0,
+					nil, api.BuildResourceList("8", "8G"), schedulingv1.PodGroupPending),
+			},
+			Queues: []*schedulingv1.Queue{
+				util.BuildQueue("c2", 1, api.BuildResourceList("4", "4G")),
+			},
+			Status: map[api.JobID]scheduling.PodGroupPhase{},
```

Comment:
What is the wanted podgroup status? Pending or Inqueue?

Reply:
case: pggroup cannot enqueue because queue resources are less than podgroup MinResources
This will be Pending.  Because JobEnqueueable Fn will be false, the status is still pending.
fixed already.